datasync/chaindatafetcher/kafka/config.go

```diff
@@ -66,6 +68,7 @@ func GetDefaultKafkaConfig() *KafkaConfig {
 		TopicResourceName:    DefaultTopicResourceName,
 		Partitions:           DefaultPartitions,
 		Replicas:             DefaultReplicas,
+		SegmentSize:          DefaultSegmentSize,
```

Comment:
How about this?

```suggestion
		SegmentSizeByte:          DefaultSegmentSizeByte,
```

Reply:
Updated with `SegmentSizeBytes` in plural. d471e8e2884ae984a4c0ef3daeb5a92546f2f2a3